client/src/components/ButtonComponent.jsx

```diff
@@ -30,20 +30,35 @@ const TextStyle = styled.span`
   line-height: 3.6rem;
   `;
 
-class ButtonContainer extends React.Component {
+class ButtonComponent extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      clicked: false,
```

Comment:
Looks like you are mainting this same state value twice, once higher up and once here. You could conditionally render the button by passing in a clicked props(already maintained at a higher level) and rendering based off its value.